codec-fix/README.md

```diff
@@ -49,5 +49,9 @@ E.g. original value: `100`, value after the removal of trailing zeroes: `1E+2`
 
 ## Release notes
 
++ 3.2.1
+  + Puts the BeginString(8) field during encode. The value to set is determined from the dictionary namespace.
+  + Verifies a message by the configured dictionary during encoding.
```

Comment:
Shouldn't we add this information to the top-level README file?

Reply:
No, because we implemented it only for FIXCodec